symplyphysics/laws/quantum_mechanics/schrodinger/time_independent_equation.py

```diff
@@ -0,0 +1,36 @@
+from sympy import Eq, sqrt, Derivative
+from sympy.physics.units import hbar
+from symplyphysics import (
+    units,
+    Symbol,
+    Function,
+    symbols,
+    clone_symbol,
+)
+
+# Description
+## The Schrödinger equation is a linear partial differential equation that governs the wave
+## function of a quantum-mechanical system.
+
+# Law: (-hbar**2 / (2 * m)) * d^2(psi(x))/dx^2 + U(x) * psi(x) = E * psi(x)
+## psi(x) - wavefunction of particle
+## x - position
```

Comment:
Does this law have 2 or 3 dimensional form?

Reply:
Yes. I'll mention the dimension in the file name